app/screens/login/form.tsx

```diff
@@ -334,7 +340,7 @@ const LoginForm = ({config, extra, serverDisplayName, launchError, launchType, l
                 endAdornment={endAdornment}
             />
 
-            {(emailEnabled || usernameEnabled) && (
+            {(emailEnabled || usernameEnabled) && config.PasswordEnableForgotLink === 'true' && (
```

Comment:
`PasswordEnableForgotLink` is a new setting present in newer servers right ? if the answer is YES, then previous server versions won't have the ability to select the option

Reply:
Ahh, good catch. I'll make sure to just check that it's not explicitly `false`